lib/mappers/bhl_mods.py

```diff
@@ -42,17 +41,14 @@ def map_description(self):
         if exists(self.provider_data, prop):
             description = []
             for s in iterify(getprop(self.provider_data, prop)):
-                try:
-                    if s.get("type") == "content":
-                        description.append(s.get("#text"))
-                except:
-                    logger.error("Error getting note/type from record %s" %
-                                 self.provider_data["_id"])
+                if "type" in s and s.get("type") == "content":
+                    description.append(s.get("#text"))
 
-            self.update_source_resource({"description": description})
+            if description:
+                self.update_source_resource({"description": description})
 
-    # OAI MODS Mapper is blowing up on this method because of arg c
-    # ount issues, even though we handle it below
+    # OAI MODS Mapper is blowing up on this method because of arg
+    # count issues,
```

Comment:
Re this code comment, `map_format()`, and `map_type()`, I'm looking at the MAPv4 crosswalk (BHL's MAPv3.1 crosswalk being for I guess what's an old Dublin Core schema, not MODS), and it simply has no mapping for dc:format, which I imagine is the ultimate reason this should be `pass`. Additionally, the crosswalk had dcterms:type mapping to `typeOfResource` but that's been struck out, so I assume that means that BHL should have neither type nor format; and `map_type()` should also just `pass`.

Confirmation from @guegueng may help here.